### PR TITLE
ENH: Update for ITKPythonPackage script updates

### PR DESCRIPTION
**Pull request by tbirdso**
Changes:
- Expose ITKPythonPackage organization and tag parameters to facilitate 1) testing ITKPythonPackage script updates from user forks and 2) applying ITKPythonPackage script patches introduced after packaging in ITKPythonBuilds archives
- Update `MANYLINUX_VERSION` parameter for Linux wheel build scripts

Depends on https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/233

### CODE REVIEW

File: .github/workflows/build-test-package-python.yml [hunk 1]
```diff
@@ -47,27 +51,27 @@ jobs:
...
+        export ITKPYTHONPACKAGE_TAG=${{ inputs.itk-python-package-tag }}
+        export ITKPYTHONPACKAGE_ORG=${{ inputs.itk-python-package-org }}
         if [ -z ${{ inputs.cmake-options }}]; then
           CMAKE_OPTIONS=""
         else
           CMAKE_OPTIONS="--cmake_options ${{ inputs.cmake-options }}"
         fi
 
-        for tarball in "-manylinux_2_28" "-manylinux2014"; do
+        for manylinux_version in "_2_28" "2014"; do
```
Comment (dzenanz) [inline .github/workflows/build-test-package-python.yml:71]: Why do we repeat the builds twice?

**Review by dzenanz — APPROVED** [review]

File: .github/workflows/build-test-package-python.yml (hunk 1 above)
Comment (tbirdso) [reply .github/workflows/build-test-package-python.yml:71]: I'm not sure I understand? Here we build once for `-manylinux_2_28` and then once for `-manylinux2014`.

Comment (dzenanz) [reply .github/workflows/build-test-package-python.yml:71]: Yes, but why do we build for two Linux versions?

Comment (tbirdso) [reply .github/workflows/build-test-package-python.yml:71]: I believe these are for compatibility with different toolchains, i.e. `-manylinux2014` for older platforms and `-manylinux_2_28` for newer platforms. See the compliance chart in the `manylinux` README: https://github.com/pypa/manylinux#manylinux

@thewtex may be able to provide more detail on the motivation for `-manylinux2014` wheels. I will also update this list in the near future for a third Linux build targeting ARM architectures to build on @thewtex 's recent work in ITKPythonPackage.

**Comment by tbirdso** [comment]
Moving forward with merging, only change from review was updating ITKPythonPackage hash to reference merge commit in `master` branch rather than dev branch

**Pull request merged**